vtipnik.py

```diff
@@ -8,7 +9,7 @@ def nacti_vtip(cesta_ke_vtipu):
     tak vrátí text vtipu jako řetězec.
     """
     try:
-        with open(cesta_ke_vtipu, 'r', encoding="utf-8") as f:
+        with open(cesta_ke_vtipu, 'r', encoding = "utf-8") as f:
```

Comment:
Tady mezery nepatří, jedná se o pojmenovaný argument nikoli o přiřazení, viz https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

```suggestion
        with open(cesta_ke_vtipu, 'r', encoding="utf-8") as f:
```